variables.tf

```diff
@@ -57,6 +57,12 @@ variable "instance_volume_size" {
   default     = 30
 }
 
+variable "imdsv2" {
```

Comment:
Nit :I think it may be a bit more descriptive to label this `require_imdsv2`?

Reply:
Agreed!